samcli/lib/providers/sam_stack_provider.py

```diff
@@ -214,3 +271,13 @@ def find_root_stack(stacks: List[Stack]) -> Stack:
             stacks_str = ", ".join([stack.stack_path for stack in stacks])
             raise ValueError(f"{stacks_str} does not contain a root stack")
         return candidates[0]
+
+    @staticmethod
+    def merge_parameter_overrides(
+        parameter_overrides: Optional[Dict], pseudo_parameter_overrides: Optional[Dict]
+    ) -> Optional[Dict]:
+        if not parameter_overrides and not pseudo_parameter_overrides:
+            return None
+        merged_parameter_overrides = parameter_overrides or {}
+        merged_parameter_overrides.update(pseudo_parameter_overrides or {})
```

Comment:
Can we make sure in class parameters documentation about which parameter has the higher precedence?

Reply:
Good question, they are not supposed to have the same keys. `parameter_overrides` should only contain none-`AWS::*` parameters and `pseudo_parameter_overrides`, as today, can only have `AWS::Region`.

Reply:
Added!